src/control/server/ctl_storage_rpc.go

```diff
@@ -429,24 +429,24 @@ func metaRdbComputeSz(cs *ControlService, ei Engine, nsps []*ctlpb.ScmNamespace)
 			continue
 		}
 
-		// NOTE DAOS-14223: This metadata size calculation won't necessarily match
-		//                  the meta blob size on SSD if --meta-size is specified in
-		//                  pool create command.
-		md_size = mp.GetUsableBytes() / uint64(ei.GetTargetCount())
+		metaBytes = mp.GetUsableBytes() / uint64(ei.GetTargetCount())
+		if memRatio > 0 {
+			metaBytes = uint64(float64(metaBytes) / float64(memRatio))
```

Comment:
Not fully understanding this part. Did you mean to multiply by memRatio rather than divide? Or is the intention to make metaBytes larger?

Reply:
Yes, the intention is to use MemRatio fraction to project the effective meta-blob (per-target) by dividing the VOS-file size by the fraction. In MD-on-SSD phase-1 metaBytes == scmBytes (VOS-file size) . I will add a comment in the subsequent PR.

Reply:
Ah, understood. Thanks for the explanation. I think a comment in this area will be helpful.

Reply:
comment added